test/suites/device-onvif-camera/main_test.go

```diff
@@ -14,15 +14,13 @@ const (
 	deviceOnvifCameraApp         = "device-onvif-camera"
 	deviceOnvifcameraService     = deviceOnvifcameraSnap + "." + deviceOnvifCameraApp
 	deviceOnvifcameraServicePort = "59984"
-	testOnvifCredFile            = "test-onvif-credentials.json"
+	testOnvifCredFile            = "/var/snap/edgex-device-onvif-camera/current/device-onvif-camera/onvif-credentials.json"
```

Comment:
`$SNAP_DATA/device-onvif-camera` is a special directory created and used for the edgex-secretstore-token interface.

Your file should be under `$SNAP_DATA/config/device-onvif-camera`
```suggestion
	testOnvifCredFile            = "/var/snap/edgex-device-onvif-camera/current/config/device-onvif-camera/test-onvif-credentials.json"
```

Reply:
It works with `sudo go test`.

Reply:
How about executing a `sudo cp` command?

Reply:
It looks like the device service has an API to submit secrets: https://github.com/edgexfoundry/device-onvif-camera/issues/101#issuecomment-1175339421